io.catenax.generic.digital_product_passport/5.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1164 @@
+
+:importer a samm:Property ;
+   samm:preferredName "Importer"@en ;
+   samm:description "Information regarding the importer of the product, if different from the owner of the passport. This attribute is mentioned in the ESPR provisional agreement from January 9th, 2024 Annex III:\n(j) information related to the importer, including the information referred to in Article 23(3) and its EORI number;\nArticle 23 states:\n(3) Importers shall, for products covered by a delegated act adopted pursuant to Article 4, indicate their name, registered trade name or registered trade mark and the postal address and electronic means of communication, where they can be contacted:  (a) on the public part of the product passport, when applicable, and\n(b) on the product or, where this is not possible, on the packaging, or in a document accompanying the product."@en ;
+   samm:characteristic :ImporterCharacteristic .
+
+:otherOperatorId a samm:Property ;
+   samm:preferredName "Other Operator Id"@en ;
+   samm:description "Identifier of the other operator. This can be a BPN."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BPNL0123456789XX" .
+
+:otherOperatorRole a samm:Property ;
+   samm:preferredName "Other Operator Role"@en ;
+   samm:description "Role of the other operator."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "distributor" .
+
+:sparePartSources a samm:Property ;
+   samm:preferredName "Spare Part Sources"@en ;
+   samm:description "Sources of possible spare parts."@en ;
+   samm:characteristic :SourcesList .
+
+:sparePart a samm:Property ;
+   samm:preferredName "Spare Part"@en ;
+   samm:description "Possible spare parts of the product."@en ;
+   samm:characteristic :PartList .
+
+:lifeUnit a samm:Property ;
+   samm:preferredName "Life Unit"@en ;
+   samm:description "The unit of the respective lifespan expressed through the possible units day, month, cycle, year and runningOrOperatingHour."@en ;
+   samm:see <https://eclipse-esmf.github.io/samm-specification/2.1.0/index.html> ;
+   samm:characteristic :LifeEnumeration ;
+   samm:exampleValue "unit:month" .
+
+:lifeValue a samm:Property ;
+   samm:preferredName "Life Value"@en ;
+   samm:description "The value as an integer for the respective lifespan."@en ;
+   samm:characteristic :LifeValueCharacteristic ;
+   samm:exampleValue 36 .
+
+:lifeType a samm:Property ;
+   samm:preferredName "Life Type"@en ;
+   samm:description "The type of lifespan represented with the values guaranteed lifetime, technical lifetime and mean time between failures. This attribute is mentioned in the ESPR proposal from March 30th, 2022 ANNEX I:\n(a) durability and reliability of the product or its components as expressed through the product's guaranteed lifetime, technical lifetime [or] mean time between failures [...]."@en ;
+   samm:characteristic :LifeTypeEnumeration ;
+   samm:exampleValue "guaranteed lifetime" .
+
+:width a samm:Property ;
+   samm:preferredName "width"@en ;
```

Comment:
Width should start with capital letter.
please check preferredName  below also for few fields